manifest.yaml

```diff
@@ -1,7 +1,9 @@
 id: lightning-jet 
 title: Lightning Jet
-version: 1.4.3
-release-notes: Initial Release to EmbassyOS v0.3.0.3
+version: 1.4.4
+release-notes: |
+  * Update to Jet v1.4.4-embassy
```

Comment:
What is "Jet v1.4.4-embassy"?

Reply:
this is the latest release tag from the upstream repo. 
https://github.com/itsneski/lightning-jet/releases/tag/v1.4.4-embassy


it seems many of your questions during PR reviews can be answered by looking at the upstream repo. is there additional detail I should be putting in this PR to pro-actively give you this info?

Reply:
Yes, please pull in/link any relevant info from upstream. 

First, I don't want to be having to search through the upstream repo for relevant info. 
Second, I've never seen an upstream repo release a version tagged with `-embassy`, so thanks for linking that; I definitely wouldn't have thought to search around the upstream repo for that.

Reply:
Probably just include that link in the release notes, so people aren't confused